wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/SyntaxRules.kt

```diff
@@ -0,0 +1,26 @@
+package com.squareup.wire.schema
```

Comment:
nit: SQ copyright on all open source .kt and .java files

Reply:
Oooops! Sorry about that!